core/trino-main/src/test/java/io/trino/execution/buffer/TestPagesSerde.java

```diff
@@ -146,8 +148,8 @@ private void testRoundTrip(List<Type> types, List<Page> pages, int blockSizeInBy
     private void testRoundTrip(List<Type> types, List<Page> pages, boolean compressionEnabled, boolean encryptionEnabled, int blockSizeInBytes)
     {
         Optional<SecretKey> encryptionKey = encryptionEnabled ? Optional.of(createRandomAesEncryptionKey()) : Optional.empty();
-        PageSerializer serializer = new PageSerializer(blockEncodingSerde, compressionEnabled, encryptionKey, blockSizeInBytes);
-        PageDeserializer deserializer = new PageDeserializer(blockEncodingSerde, compressionEnabled, encryptionKey, blockSizeInBytes);
+        PageSerializer serializer = new PageSerializer(blockEncodingSerde, compressionEnabled, new Lz4Compressor(), encryptionKey, blockSizeInBytes);
```

Comment:
It'd be nice to test all supported compressors with PageSerializer

Reply:
> It'd be nice to test all supported compressors with PageSerializer

@wendigo I have added tests about all compressors, they seemed passed, please cc again~